makefile

```diff
@@ -1113,11 +1117,15 @@ endef
 $(foreach t,$(releasetbb.LIBS_Y),$(eval $(call .release.t,$t,$(RELEASEDIR.tbb.soia))))
 $(foreach t,$(releasetbb.LIBS_A),$(eval $(call .release.t,$t,$(RELEASEDIR.tbb.libia))))
 
+#----- cmake configs generation
+_release_cmake_configs:
+	$(if $(shell bash -c "command -v cmake"),cmake -DINSTALL_DIR=$(RELEASEDIR.lib)/cmake/oneDAL -P cmake/scripts/generate_config.cmake,echo 'cmake configs generation skipped')
```

Comment:
cmake generation is part of internal infra currently - corresponding changes should be made there

Reply:
Is there any problem if configs are generated twice? I think auto generation of configs will be more useful for users

Reply:
Agree that it should be generated here. I'm not sure what would happen on second call - if it works - then fine

Reply:
It works, same configs are generated each time